src/inputs/checkbox-radio.scss

```diff
@@ -114,8 +121,8 @@
   // Adds focus to enabled checkbox & radio
   input:enabled {
     &:focus {
-      ~ .iui-checkbox-checkmark,
-      ~ .iui-radio-dot {
+      ~ .iui-checkbox-checkmark::after,
+      ~ .iui-radio-dot::after {
```

Comment:
`::after` should not be needed for focus outline

Reply:
Without `::after` the focus does not have rounded corners:
![Screen Shot 2022-01-07 at 1 51 10 PM](https://user-images.githubusercontent.com/849817/148592412-0ba24463-e497-4b9b-8ccc-a898a9a9b3f3.png)

However, I can fix that by doing this:
```scss
.iui-checkbox-checkmark,
.iui-checkbox-checkmark::after {
  border-radius: $iui-border-radius;
}
```
&
```scss
.iui-radio-dot,
.iui-radio-dot::after {
  border-radius: 50%;
}
```

Reply:
Your suggestion below solves this, is working now.